pkg/proxy/ipvs/proxier_test.go

```diff
@@ -2897,10 +3127,14 @@ func checkIPSet(t *testing.T, fp *Proxier, ipSet netlinktest.ExpectedIPSet) {
 			t.Errorf("Check ipset entries failed for ipset: %q, expect %d, got %d", set, len(entries), len(ents))
 			continue
 		}
-		if len(entries) == 1 {
-			if ents[0] != entries[0].String() {
-				t.Errorf("Check ipset entries failed for ipset: %q", set)
+		for _, entry := range entries {
```

Comment:
Wondering if we can just `reflect.DeepEqual` the two ipset entries here and then `t.Errorf` if not equal, or does that fail for other reasons?

Reply:
> Wondering if we can just `reflect.DeepEqual` the two ipset entries here and then `t.Errorf` if not equal, or does that fail for other reasons?

we can not use deepEqual, because the return of ipset.ListEntries does not guarantee any order of elements in the slice, which may not match with the order of expected entries. And Array values are deeply equal when their corresponding elements are deeply equal.

Reply:
Sort then DeepEqual? Or update the fake `ListEntries` to use `List()` instead of `UnsortedList`.

Reply:
> 
> 
> Sort then DeepEqual? Or update the fake `ListEntries` to use `List()` instead of `UnsortedList`.

Thanks for review, Added `Sort then DeepEqual`